### PR TITLE
make sure to install the same version of golang across scripts

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: hack/jenkins/installers/check_install_golang.sh [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
   exit 1
 fi
 
-VERSION_TO_INSTALL=${1}
-INSTALL_PATH=${2}
+VERSION_TO_INSTALL=$(grep '^GO_VERSION' Makefile | awk '{ print $3 }')
+INSTALL_PATH=${1}
 
 function current_arch() {
   case $(arch) in
```
Comment (spowelljr) [inline hack/jenkins/installers/check_install_golang.sh:29]: I believe line 70 & 71 in this file need to be updated as well

Comment (spowelljr) [reply hack/jenkins/installers/check_install_golang.sh:29]: Never mind, read it wrong

**Review by spowelljr — APPROVED** [review]

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
ok-to-build-image

File: hack/jenkins/common.sh [hunk 2]
```diff
@@ -91,7 +91,7 @@ else
 fi
 
 # installing golang so we could do go get for gopogh
-./installers/check_install_golang.sh "1.17" "/usr/local" || true
```
Comment (medyagh) [inline hack/jenkins/common.sh:94]: this will need to be updated in our Update Golang automation too

**Review by medyagh — CHANGES_REQUESTED** [review]

File: hack/jenkins/common.sh (hunk 2 above)
Comment (sharifelgamal) [reply hack/jenkins/common.sh:94]: done, removed from update_golang_version.go

**Review by medyagh — APPROVED** [review]

**Pull request merged**